README.md

```diff
@@ -246,14 +242,10 @@ e) składanie sprawozdań ze swojej działalności na Walnym Zebraniu Członków
 
 f) zatwierdzanie rocznego sprawozdania finansowego.
 
-W razie gdy skład władz Stowarzyszenia ulegnie zmniejszeniu w czasie trwania kadencji, uzupełnienie ich składu może nastąpić w drodze kooptacji spośród członków zwyczajnych Stowarzyszenia, której dokonują pozostali członkowie organu, który uległ zmniejszeniu. W tym trybie można powołać nie więcej niż połowę składu organu.
-
-
+W razie, gdy skład władz wybieralnych Stowarzyszenia ulegnie zmniejszeniu w czasie trwania kadencji, uzupełnienie ich składu może nastąpić w drodze kooptacji spośród członków zwyczajnych Stowarzyszenia, której dokonują pozostali członkowie organu, który uległ zmniejszeniu. W tym trybie można powołać nie więcej, niż połowę składu organu.
```

Comment:
@kpc21 tbh ciągle nie kumam tej części. przydałby mi się przykład - nie w statucie, ale gdzieś na boku

Reply:
Zarząd ma 3 członków: ciebie, Aleksego i Kacpra Szmigla (tak jest w tymczasowym, to posłużę się takim przykładem). Chcesz zrezygnować. Zarząd musi mieć minimum trzech członków. Organem, który uległ zmniejszeniu jest zarząd. Zarząd musi dokooptować kogoś spośród członków zwyczajnych stowarzyszenia. Czyli Aleksy i Kacper typują kogoś spośród członków, ewentualnie odpytują, kto by chciał wejść. Zgłasza się Księżak (albo typują go, a on się zgadza) i bez głosowania uzupełnia on skład zarządu.

Inaczej będzie, jeśli zrezygnujesz np. ty i Aleksy. Kacper nie może sobie dokooptować losowego członka – muszą odbyć się przedterminowe wybory.